SPECS/node-problem-detector/002-add_mariner_OSVersion.patch

```diff
@@ -1,12 +1,12 @@
 diff -Naur a/pkg/util/helpers_linux.go b/pkg/util/helpers_linux.go
---- a/pkg/util/helpers_linux.go	2021-05-14 17:05:59.000000000 -0700
-+++ b/pkg/util/helpers_linux.go	2021-06-15 12:26:43.919828110 -0700
-@@ -57,6 +58,8 @@
+--- a/pkg/util/helpers_linux.go	2024-02-28 17:22:37.000000000 +0000
++++ b/pkg/util/helpers_linux.go	2024-03-19 00:22:29.127451201 +0000
+@@ -58,6 +58,8 @@
  		return getDebianVersion(osReleaseMap), nil
  	case "rhel":
  		return getDebianVersion(osReleaseMap), nil
 +	case "mariner":
 +		return getDebianVersion(osReleaseMap), nil
- 	default:
```

Comment:
is this fully removing the default: case?

Reply:
Nope, these lines in the patch are being removed because default: is now well below us in the switch-case.